lib/taskana-data/src/main/resources/sql/sample-data/history-event.sql

```diff
@@ -1,52 +1,47 @@
-('BPI:06'               ,'BPI:05',                  'TKI:000000000000000000000000000000000001',     'UPDATE', RELATIVE_DATE(0) ,                         'USER_2_1', 'DOMAIN_A',        'WBI:100000000000000000000000000000000002', '11'                , ''            , '22'                  , ''            , ''            , 'L140101'                             , 'TASK'                                                ,'DOCTYPE_DEFAULT'                              , 'created a bug'                               ,''                     ,''                     ,'custom1'      ,''                     , 'custom3'     ,''                     ,'123'          ,'555'),
-('BPI:02'               ,''             ,                   'TKI:000000000000000000000000000000000000',     'DELETE',   DATEADD('DAY', -1, RELATIVE_DATE(0) ),  'USER_1_2', 'DOMAIN_B',         'WBI:100000000000000000000000000000000001', '00'                , 'PASystem', '00'                      , 'VNR'         , '11223344', ''                                        , ''                                                    ,''                                                             , 'created by Peter'                    ,'2old_val'     ,'new_val2'     ,'custom1'      ,''                     , 'custom2'     ,''                     ,'234'          ,'777');
+INSERT INTO HISTORY_EVENTS (BUSINESS_PROCESS_ID, PARENT_BUSINESS_PROCESS_ID, TASK_ID, EVENT_TYPE, CREATED, USER_ID, DOMAIN, WORKBASKET_KEY, POR_COMPANY, POR_SYSTEM, POR_INSTANCE, POR_TYPE, POR_VALUE, TASK_CLASSIFICATION_KEY, TASK_CLASSIFICATION_CATEGORY, ATTACHMENT_CLASSIFICATION_KEY, OLD_VALUE, NEW_VALUE, CUSTOM_1, CUSTOM_2, CUSTOM_3, CUSTOM_4, DETAILS) VALUES
+-- BUSINESS_PROCESS_ID, PARENT_BUSINESS_PROCESS_ID, TASK_ID,                                        EVENT_TYPE, CREATED,                                        USER_ID,    DOMAIN,         WORKBASKET_KEY,                                                     POR_COMPANY     , POR_SYSTEM, POR_INSTANCE      , POR_TYPE      , POR_VALUE     , TASK_CLASSIFICATION_KEY, TASK_CLASSIFICATION_CATEGORY , ATTACHMENT_CLASSIFICATION_KEY , OLD_VALUE     , NEW_VALUE     , CUSTOM_1      , CUSTOM_2      , CUSTOM_3      , CUSTOM_4
+('BPI:01'               ,'',                        'TKI:000000000000000000000000000000000000',     'CREATE',   CURRENT_TIMESTAMP ,                      'USER_2_2', 'DOMAIN_B',         'WBI:100000000000000000000000000000000001', '00'                , 'PASystem', '00'                      , 'VNR'         , '11223344'                     ,' L140101'                         , 'TASK'                       ,''                    ,'old_val12'      ,'new_val12'      ,'custom1'      ,'custom2'      , 'custom3'     ,'custom4', 'some Details'),
```

Comment:
I'm not sure but haven't you changed the type to 'TASK_CREATED' instead of 'CREATE'? Did I get this wrong?

Reply:
Yes, this is old Test-data. I adjusted it now